extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
@macbre Can you double check the varnish config?  I can confirm what @jsutterfield has said.  Here's the AJAX request URL that I'm using:
```bash
curl -v -x ap-s23:80 "http://garth.wikia.com/index.php?action=ajax&article=2460&convertToFormat=&id=2479&method=axEdit&rs=ArticleCommentsAjax&useskin=oasis"
```
1) Via debugging on a sandbox I can confirm that this request hits this line of code: https://github.com/Wikia/app/blob/dev/includes/AjaxResponse.php#L130
which sets the correct headers.
2) Via this curl command to the backend on ```ap-s23```, I can confirm that this produces a ```Cache-Control``` with ```no-cache```, an ```Expires``` header in the past and no ```X-Pass-Cache-Control``` to be found:
```
< HTTP/1.1 200 OK
< Date: Thu, 30 Jul 2015 21:49:23 GMT
* Server Apache is not blacklisted
< Server: Apache
< X-Content-Type-Options: nosniff
< Status: 200 OK
< Last-Modified: Thu, 30 Jul 2015 21:49:23 GMT
< Expires: Mon, 26 Jul 1997 05:00:00 GMT
< Cache-Control: no-cache, must-revalidate
< Pragma: no-cache
< Vary: Accept-Encoding
< Content-Length: 46
< Content-Type: application/json; charset=utf-8
```
3) When I remove the ```-x``` option to allow it to hit varnish, the ```Cache-Control``` header no longer has ```no-cache``` and there is no longer an ```Expires``` header:
```
< HTTP/1.1 200 OK
* Server Apache is not blacklisted
< Server: Apache
< X-Content-Type-Options: nosniff
< Status: 200 OK
< Last-Modified: Thu, 30 Jul 2015 21:57:40 GMT
< Content-Type: application/json; charset=utf-8
< X-Cacheable: YES
< Content-Length: 46
< Accept-Ranges: bytes
< Date: Thu, 30 Jul 2015 21:57:40 GMT
< Age: 0
< Connection: keep-alive
< X-Served-By: cache-wk-sjc3160-WIKIA, cache-sjc3134-SJC
< X-Cache: MISS, MISS
< X-Cache-Hits: 0, 0
< X-Timer: S1438293460.059463,VS0,VE113
< Vary: Accept-Encoding
< Set-Cookie: wikia_beacon_id=sMNTnjZbJC; domain=.wikia.com; path=/; expires=Tue, 26 Jan 2016 21:57:40 GMT;
< Set-Cookie: Geo={%22city%22:%22FIXME%22%2C%22country%22:%22US%22%2C%22continent%22:%22NA%22}; path=/
< Cache-Control: private, s-maxage=0, max-age=0, must-revalidate
```
I've retried the curl request several times and I always get new content every 2 minutes